warp/backend.go

```diff
@@ -81,6 +84,11 @@ func (b *backend) AddMessage(unsignedMessage *avalancheWarp.UnsignedMessage) err
 	return nil
 }
 
+func (b *backend) CacheMessage(unsignedMessage *avalancheWarp.UnsignedMessage) {
+	b.messageCache.Put(unsignedMessage.ID(), unsignedMessage)
+	log.Debug("Adding warp message to cache", "messageID", unsignedMessage.ID())
+}
+
 func (b *backend) GetSignature(messageID ids.ID) ([bls.SignatureLen]byte, error) {
```

Comment:
I guess we don't need to modify this, despite the issue description? https://github.com/ava-labs/subnet-evm/issues/729:
`Modify GetSignature(...) to check the in-memory map for an eligible warp message, so that it will sign these as well`

Reply:
Yes, GetSignature first checks the signature cache, then calls `GetMessage` which checks the message cache and returns immediately. There was no need to modify it.